### ISSUE TITLE
Accessibility policy: actions to improve our quality process

### ISSUE THREAD

**Issue by florian-sanders-cc** [CleverCloud/clever-components#834]
## Context

While writing about the component accessibility policy, it became clear that it's time to improve the way we handle this subject.

We have good intentions, a skilled team but we lack tools and process to make sure everything has been checked properly.

We cannot conduct a WCAG audit for every new Web Component but we can definitely review some essential points.

TODO:

- [ ] add a checklist to be used to check new components before they are merged. (zoom, text resizing?, ARIA misuse, element accessible names, etc.)
- [ ] create a template for a11y issues. If we want people to be able to point out a11y issue, we should ease the process by providing a template with useful infos to fill in: component impacted, OS, screen reader, browser, user impact, WCAG SC if any, etc.).
- [ ] ...
